lib/app.js

```diff
@@ -31,18 +33,19 @@ module.exports = settings => {
 
   app.use(content);
 
+  app.use(urls.dashboard, dashboard());
```

Comment:
One small change... this line wants to be moved down to the bottom of the routing stack. as it stands, `app.use('/',...)` will hit _every_ other url along the way too, which isn't necessarily terrible, but it's inefficient, so try to order routes from most to least specific, and since `/` is the least specific possible it wants to go at the bottom.

Reply:
I did this myself because #livingdangerously